manifests/init.pp

```diff
@@ -113,8 +110,46 @@
   $wsgi_threads      = $sentry::params::wsgi_threads,
 ) inherits ::sentry::params {
 
+  class { '::sentry::setup':
```

Comment:
Can these class declarations be removed in favor of the `contain` statements plus chaining arrows?